src/Symfony/Bridge/PhpUnit/bin/simple-phpunit

```diff
@@ -117,7 +117,11 @@ if (!file_exists("$PHPUNIT_DIR/phpunit-$PHPUNIT_VERSION/phpunit") || md5_file(__
         'requires' => array('php' => '*'),
     );
 
-    if (1 === count($info['versions'])) {
+    $stableVersions = array_filter($info['versions'], function($v) {
+        return 0 === preg_match('/-dev|dev-/', $v);
```

Comment:
`return !preg_match('/-dev|dev-/', $v);`
or maybe
`return !preg_match('/-dev$|^dev-/', $v);`?

Reply:
The strict comparison is clearer IMO.

Reply:
I personally prefer the strictness but don't think it's entirely necessary in this particular case.

@nicolas-grekas I've amended and force-pushed my commit with your requested changes.